ui/src/main/java/edu/wpi/grip/ui/preview/PreviewsView.java

```diff
@@ -48,24 +58,203 @@ public PreviewsView(EventBus eventBus) {
     }
 
     @Subscribe
+    /**
+     * This function is called when a step moves in the pipeline to adjust the positions of any open previews it has
+     * to reflect the new order of the pipeline.
+     */
```

Comment:
Can you move the comment  to above the `@Subscribe` attribute. Its the code style we've been following everywhere else and here it looks out of place.